hedyorg/hedy
# [BUG FIX] Fixes single adventure language switcher

**Description**
In this PR we fix the keyword language switcher on the specific adventure mode. As we didn't have the `commands` list yet on this route the switcher was empty. We also add the correct adventure GET dependent on the keyword language to make sure this changes as well when switching.

**Fixes**
This PR fixes #3485.

**How to test**
Navigate to any specific adventure such, such as `/adventure/parrot/1`. Verify that the language switcher works and the adventure is also correctly parsed in the chosen keyword language.
